### PR TITLE
Gateway beta

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

**Comment by bshaffer** [comment]
Added missing `phpunit.xml.dist`, which is required for the tests to run on Kokoro

File: iot/test/iotTest.php [hunk 1]
```diff
@@ -269,6 +278,52 @@ public function testPatchRsa()
...
+        self::$devices[] = $deviceId;
+
+        $output = $this->runCommand('beta-bind-device-to-gateway', [
+            'registry' => self::$registryId,
+            'gateway' => $gatewayId,
+            'device' => $deviceId,
+        ]);
+        $this->assertContains('Device bound', $output);
+
+        $output = $this->runCommand('unbeta-bind-device-from-gateway', [
```
[the remark this answers is not in the document]
Comment (gguuss) [reply iot/test/iotTest.php]: Thanks for the proactive review!  Looks like iot was missing from the test config :)

**Comment by gguuss** [comment]
For posterity, [here is the issue that affects this PR](https://github.com/googleapis/google-cloud-php/issues/1583).

**Comment by gguuss** [comment]
@bshaffer I'll try and write the tests for listing devices for gateway so we'll be ready when the client library is fixed.

**Comment by gguuss** [comment]
This is working now from my machine, once the tests are green, we should be good to go.

**Pull request merged**